js/anims-missiles.js

```diff
@@ -59,41 +64,69 @@ function createMissileClickHandler() {
     var cities = document.querySelectorAll('.city')
     // forEach loop changed with Array.prototype.forEach.call due to compatibility issues in IE10
     Array.prototype.forEach.call(cities, function(city) {
-        city.addEventListener('click', function() {
-            columnID = this.getAttribute('data-city')
-            isHit()
-        })
-    })//end forEach
+        city.addEventListener('click', checkForMissiles)
+    })
 }
+
+/**
+ * This was previously an anonymous function given its own function. It sets the column id and executes the isHit()
+ * method to scan for any missiles in the column
+ */
+function checkForMissiles() {
```

Comment:
this function doesnt check for missiles, not sure the function name is correct

Reply:
fixed in eb98601